canvas/admin.py

```diff
@@ -7,4 +7,4 @@
 admin.site.register(CanvasCourseRegistration)
 admin.site.register(Event)
 admin.site.register(TokenUseOption)
-admin.site.register(TokenUse)
+admin.site.register(TokenUse)
```

Comment:
Don't randomly change files. Please take a look at your changes every time you make a PR.